api/controllers/productController.js

```diff
@@ -13,6 +13,8 @@ const multer = require("multer");
 let dirImg   = '/image/';
 
 exports.add = async (req, res) => {
+  const token = await concordium.getNextNFT();
```

Comment:
now that you have the token here, just make sure the token and block are used in https://github.com/aesirxio/web3-demo-backend/pull/8/files#diff-ccf008b2097c4d1961dd9af98911474b941d37680598698ee1ab84bd0e1ad993R34